client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -288,6 +287,16 @@ const Checkout = React.createClass( {
 			this.props.clearSitePlans( selectedSiteId );
 		}
 
+		if ( receipt && receipt.receipt_id ) {
+			const receiptId = receipt.receipt_id;
```

Comment:
It would have to be 
```
const { receipt_id: receiptId } = receipt;
```
which kind of has the same length, so I'd rather just leave it as it. :)